src/Common/Internal/Atom/AtomBase.php

```diff
@@ -300,26 +294,23 @@ protected function writeOptionalAttribute(
     /**
      * Writes the optional elements namespaces.
      *
-     * @param \XmlWriter $xmlWriter    The XML writer.
+     * @param \XMLWriter $xmlWriter    The XML writer.
      * @param string     $prefix       The prefix.
      * @param string     $elementName  The element name.
      * @param string     $namespace    The namespace name.
      * @param string     $elementValue The element value.
-     *
-     * @return none
      */
     protected function writeOptionalElementNS(
-        $xmlWriter,
+        \XMLWriter $xmlWriter,
         $prefix,
         $elementName,
         $namespace,
         $elementValue
     ) {
-        Validate::notNull($xmlWriter, 'xmlWriter');
         Validate::isString($elementName, 'elementName');
 
         if (!empty($elementValue)) {
-            $xmlWriter->writeElementNS(
+            $xmlWriter->writeElementNs(
```

Comment:
Interesting, it looks like PHPStorm has incorrect definitions. I've created a PR for PHPStorm https://github.com/JetBrains/phpstorm-stubs/pull/126